src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -184,6 +186,14 @@ class CoreWorkerDirectActorTaskSubmitter
     /// actor failure.
     uint64_t num_completed_tasks = 0;
```

Comment:
Can you update the doc for this member to explain that it only tracks consecutive tasks? Maybe we should call this something different to indicate that it's the next task counter that we're expecting.

Reply:
Added some comments but can't really think of a better name

Reply:
How about `next_task_reply_position_`?